pkg/clustertask/clustertask_test.go

```diff
@@ -91,22 +92,28 @@ func TestClusterTask_List(t *testing.T) {
 	testParams := []struct {
 		name   string
 		client *cli.Clients
+		params *test.Params
 		want   []string
 	}{
 		{
 			name:   "Single clusterTask",
 			client: c1,
+			params: p,
 			want:   []string{"clustertask"},
 		},
 		{
 			name:   "Multi clusterTasks",
 			client: c2,
+			params: p2,
 			want:   []string{"clustertask", "clustertask2"},
 		},
 	}
 
 	for _, tp := range testParams {
 		t.Run(tp.name, func(t *testing.T) {
+			if err := actions.InitializeAPIGroupRes(tp.params.Tekton.Discovery()); err != nil {
```

Comment:
Wouldn't this be initialised using `tp.params.Tekton.Discovery()` automatically?

Reply:
No, it was not getting initialized

Reply:
Same as @sthaha , `List` seems to call `actions.List` that calls `GetGroupVersionResource`, so I don't understand why we need to call `InitializeAPIGroupRes` before.

Reply:
Because it is getting initialized in running first test, then for second test it was not getting initialized as the value is already so there, so getting called from every test to have the value once.